### PR TITLE
Adds core.md web interface links 

### PR DESCRIPTION
**Pull request by chepo92**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Context: I was installing the core on a raspberry pi, as I didn't want to wipe my current installation, after solving typical intall issues with dependencies I ran hass but I wasn't able to reach the webserver on http://homeassistant.local:8123 which is the url indicated in the docs, http://homeassistant:8123 didn't work either, the correct one was  http://localhost:8123/
- It took me a while and some attemps with -v flag realizing the problem was the url I was following, as a note later says "When you run the hass command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get “site cannot be reached” error when accessing the web interface. This will only happen for the first time, and subsequent restarts will be much faster."
- Adds optionally correct link to installation when installing the core

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @chepo92, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18367).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by frenck** [comment]
Could you elaborate on this change?

**Comment by chepo92** [comment]
> 
> 
> Could you elaborate on this change?

I just edited the pr with my story in "Additional information"

File: source/_includes/installation/core.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Start Home Assistant Core for the first time. This will complete the installatio
 hass
 ```
 
-You can now reach your installation via the web interface on `http://homeassistant.local:8123`.
+You can now reach your installation via the web interface on `http://homeassistant.local:8123`, `http://localhost:8123` or `http://X.X.X.X:8123`  (replace X.X.X.X with your machines’s IP address)..
```
Comment (frenck) [inline source/_includes/installation/core.md]:
```suggestion
You can now reach your installation via the web interface on `http://homeassistant.local:8123`.

If this address doesn't work you may also try `http://localhost:8123` or `http://X.X.X.X:8123` (replace X.X.X.X with your machines’ IP address).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @chepo92 👍

**Pull request merged**